src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -2396,6 +2396,11 @@ public static string RenameObservedData(string oldName, string newName)
          return $"Renamed observed data from '{oldName}' to '{newName}'";
       }
 
+      public static string RenameSimulation(string oldName, string newName)
+      {
+         return $"Renamed simulation from '{oldName}' to '{newName}'";
+      }
+
```

Comment:
I'd change that to an expression body. It was probably written before that feature was available.